client/packages.go

```diff
@@ -117,3 +118,27 @@ func (client *Client) Snap(name string) (*Snap, error) {
 
 	return pkg, nil
 }
+
+// RemoveSnap removes the snap with the given name
+func (client *Client) RemoveSnap(name string) error {
```

Comment:
I'm curious about the return here, given that this is a async call, would it make sense to return (asyncStatusURL string, err error) here? And then have some client function that can read the that async operation url? Or is this all planed already (or done differently :) ?